app/models/user.rb

```diff
@@ -4,6 +4,9 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
+  attribute :full_name, :string, default: ""
+  attribute :description, :string, default: ""
```

Comment:
These two helper invocations should be superfluous, because Rails will automatically load the attributes from the DB Schema.

According to the official documentation (https://api.rubyonrails.org/classes/ActiveRecord/Attributes/ClassMethods.html#method-i-attribute) this method is for *overriding* this behaviour, which we do not want to do here, because the default behaviour should be just fine.

Reply:
We added these fields to have as documentation inside of the `user.rb` class, where we'll need to refer to them often. What do you think about having these lines as comments instead?